### DISCUSSION TITLE
HmIP-HAP als Hauptsender/-empfänger?

### DISCUSSION THREAD

**Discussion by Tomson124** [OpenCCU/OpenCCU#1837] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen, 

ist es möglich den Acess Point als Hauptemfänger und -sender zu nutzen? Da bei mir RasperryMatic aufs QNAP soll und das wiederum in einem Netzwerkschrank aus Metall steht, ist die Reichweite des USB-Sticks vermutlich miserabel.
Sofern ich es verstanden habe brauche ich den Stick ja so oder so, egal ob ich die HmIP-HAP als Repeater nutze oder nicht, richtig?
Vielen Dank.

**Comment by jens-maus** [comment, marked as answer]
> ist es möglich den Acess Point als Hauptemfänger und -sender zu nutzen? 

Nein, das geht nicht. homematicIP kennt kein Hauptempfänger/Nebenempfänger oder Sender. Das passiert alles vollautomatisch und völlig autark. Das einzige was du machen kannst ist die Antenne des HmIP-RFUSB komplett hardwaretechnisch abzuschneiden und ggen den RFUSB in eine Metallbox zu verbannen, sodass das Empfangen+Senden hiermit nicht mehr geht. Dann sollte theoretisch hier dann der HmIP-HAP die Aufgabe übernehmen weil der RFUSB selbst ja dann eine schlechte Sende/Empfangsqualität aufweist und die homematicIP Logik dann den HAP als AccessPoint verwenden wird/muss. Aber ein Garant darauf gibt es nicht.

**Comment by Baxxy13** [comment]
Ich würde den Stick (HmIP-RFUSB) zuerst mal mit einer guten USB-Verlängerung aus dem Netzwerkschrank herausführen. 
2 "Empfänger" sind doch besser als einer. 
Wenn es dann Probleme mit Kommunikationsstörungen gibt kannst du den immer noch "kastrieren" und abschirmen.
